src/Compilers/Core/Portable/CommandLine/CommonCommandLineArguments.cs

```diff
@@ -279,6 +279,40 @@ internal CommandLineArguments()
         {
         }
 
+        internal static string NormalizePathPrefix(string filePath, ImmutableArray<KeyValuePair<string, string>> pathMap)
```

Comment:
I think the dependency of SourceFileResolver on CommandLineArguments is weird. Why isn't PathUtilities a good place?

---
In reply to: [117100409](https://github.com/dotnet/roslyn/pull/19593#discussion_r117100409) [](ancestors = 117100409)